spring-ai-core/src/main/java/org/springframework/ai/vectorstore/filter/converter/PrintFilterExpressionConverter.java

```diff
@@ -0,0 +1,31 @@
+package org.springframework.ai.vectorstore.filter.converter;
+
+import org.springframework.ai.vectorstore.filter.Filter.Expression;
+import org.springframework.ai.vectorstore.filter.Filter.Group;
+import org.springframework.ai.vectorstore.filter.Filter.Key;
+
+//
+// Test, print walker
+//
+public class PrintFilterExpressionConverter extends AbstractFilterExpressionConverter {
```

Comment:
javadoc - do end users need this?  for debugging?

Reply:
+